solvers/qpSpline/PiecewisePolynomial.cpp

```diff
@@ -0,0 +1,116 @@
+#include "PiecewisePolynomial.h"
+#include <stdexcept>
+#include <cassert>
+#include <algorithm>
+
+using namespace std;
+
+PiecewisePolynomial::PiecewisePolynomial(std::vector<Polynomial> const& polynomials, std::vector<double> const& segment_times) :
```

Comment:
As much as you won't like this: I would consider renaming it PPTrajectory.  I'm happy to discuss renaming both eventually... but I just want them to be logically connected.

On Apr 10, 2015, at 8:34 AM, Twan Koolen <notifications@github.com> wrote:

> In solvers/qpSpline/PiecewisePolynomial.cpp:
> 
> > @@ -0,0 +1,116 @@
> > +#include "PiecewisePolynomial.h"
> > +#include <stdexcept>
> > +#include <cassert>
> > +#include <algorithm>
> > +
> > +using namespace std;
> > +
> > +PiecewisePolynomial::PiecewisePolynomial(std::vector<Polynomial> const& polynomials, std::vector<double> const& segment_times) :
> What does this mean concretely right now, do I have to rename or add anything? There are a lot missing methods compared to PPTrajectory of course. I can fill those in over time.
> 
> Yeah, systems/trajectories is a better place and it won't mess with Robin's stuff so I'll move it.
> 
> —
> Reply to this email directly or view it on GitHub.
>